x-pack/plugins/endpoint/public/embeddables/resolver/store/data/selectors.ts

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createSelector } from 'reselect';
+import { ResolverState } from '../../types';
+import { sampleData } from './sample';
+import { depthFirstPreorder, levelOrder } from '../../lib/tree_sequencers';
+import { Vector2 } from '../../types';
+import { add as vector2Add } from '../../lib/vector2';
+
+const unit = 100;
+const distanceBetweenNodesInUnits = 1;
+const distanceBetweenNodes = distanceBetweenNodesInUnits * unit;
+
+function dataSelector(state: ResolverState) {
+  return sampleData.data.result.search_results;
+}
+
+export function isGraphableProcess(event) {
```

Comment:
:question: If we ever get to the point where we want to graph things other than processes, we could change the name of this.